plenum/server/consensus/view_change_service.py

```diff
@@ -56,49 +146,75 @@ def start_view_change(self):
         )
         self._network.send(vc)
 
+        self._stasher.unstash()
```

Comment:
Currently the only reason for stashing messages in view change service is message containing future view no, so no need for introducing extra parameters. When we actually need more reasons for stashing/unstashing it can be done in the following way:
```
STASH_CATCHUP = STASH + 0
STASH_FUTURE_VIEW = STASH + 1

def _validate(msg):
   ...
   return STASH_FUTURE_VIEW
   ...

def start_view_change():
    ...
    self._stasher.unstash(STASH_FUTURE_VIEW)
```

Reply:
But what about receiving ViewChange messages during Catchup? I think we need to stash it as well, so we have at least 2 cases already...